docs/reference/ImageFilter.rst

```diff
@@ -37,12 +37,13 @@ image enhancement filters:
 * **SMOOTH**
 * **SMOOTH_MORE**
 
-.. autoclass:: PIL.ImageFilter.GaussianBlur
+.. autoclass:: PIL.ImageFilter.Color3DLUT
 .. autoclass:: PIL.ImageFilter.BoxBlur
-.. autoclass:: PIL.ImageFilter.UnsharpMask
+.. autoclass:: PIL.ImageFilter.GaussianBlur
```

Comment:
Oh, I tried to order classes alphabetically and was happy that BoxBlur and GaussianBlur come together because they are very similar. But now I see that actually, Color3DLUT should be between them ) So it's still not alphabetically.

Also, it better to return RankFilter and UnsharpMask on its place.